static/app/views/settings/projectDebugFiles/sources/customRepositories/index.tsx

```diff
@@ -184,8 +184,11 @@ function CustomRepositories({
           method: 'POST',
         }
       );
+      addSuccessMessage(t('Request to sync repository sent.'));
```

Comment:
nit: I'd write "Repository sync started" but I'm not sure what tone of voice guidelines there are so maybe I'm talking nonsense.